README.md

```diff
@@ -31,7 +31,59 @@ The file `config.json` should contain your train_name and sciper.
 
 You cannot use the same sciper or train_name as another player already connected.
 
-### 3. Run the client
+```json
+{
+    "sciper": "000000",
+    "train_name": "Player"
+}
+```
+
+You cannot use the same sciper or train name as another player already connected.
+
+### 3. Game Modes
+
+The game supports two different modes that can be set in the `config.json` file:
+
+- **Competitive Mode** (`"game_mode": "competitive"`): In this mode, the client connects to a remote server to compete against other players' agents in a battle. The client uses the agent specified in the `competitive_agent` field of the configuration file.
+
+- **Local Evaluation Mode** (`"game_mode": "local_evaluation"`): This mode allows you to run a local evaluation by loading multiple agents from your machine. The agents specified in the `local_agents` list in `config.json` will compete against each other, allowing you to test and compare different versions of your agents. For better organization, it's recommended to store your agents in the "agents" folder.
+
+Example configuration in `config.json`:
+```json
+{
+    "game_mode": "competitive",  // or "local_evaluation"
```

Comment:
please don't put invalid json snippets in the README. If you make game_mode an enum, students can see the valid values.